src/components/tasks/card/ConditionalLinkWrapper.tsx

```diff
@@ -21,7 +21,11 @@ export const ConditionalLinkWrapper: FC<ConditionalLinkWrapperProps> = ({
                     pathname: redirectingPath,
                 }}
                 as={`/tasks/${taskId}`}
-                style={{ textDecoration: 'none' }}
+                style={{
+                    textDecoration: 'none',
+                    overflow: 'hidden',
+                    textOverflow: 'ellipsis',
+                }}
```

Comment:
Why you adding the css in .tsx file ?

Reply:
Done with changes @sahsisunny...Can you check my latest commit